modules/alloy/src/main/ts/ephox/alloy/keying/TabbingTypes.ts

```diff
@@ -112,11 +112,12 @@ const create = (cyclicField: FieldProcessor): KeyingType.KeyingType<TabbingConfi
   const getKeydownRules = Fun.constant([
     KeyRules.rule(KeyMatch.and([ KeyMatch.isShift, KeyMatch.inSet(Keys.TAB) ]), goBackwards),
     KeyRules.rule(KeyMatch.inSet(Keys.TAB), goForwards),
-    KeyRules.rule(KeyMatch.inSet(Keys.ESCAPE), exit),
     KeyRules.rule(KeyMatch.and([ KeyMatch.isNotShift, KeyMatch.inSet(Keys.ENTER) ]), execute)
   ]);
 
-  const getKeyupRules = Fun.constant([ ]);
+  const getKeyupRules = Fun.constant([
```

Comment:
This needs an Alloy changelog entry and you also need to add the minor version bump in `versisons.txt`.

- [ ] Alloy changelog
- [ ] Add version bump to `versions.txt`